src/vs/workbench/api/common/extHostDebugService.ts

```diff
@@ -945,6 +945,9 @@ export class ExtHostVariableResolverService extends AbstractVariableResolverServ
 			getConfigurationValue: (folderUri: URI | undefined, section: string): string | undefined => {
 				return configurationService.getConfiguration(undefined, folderUri).get<string>(section);
 			},
+			getAppRoot: (): string | undefined => {
+				return env ? env['VSCODE_CWD'] : undefined;
```

Comment:
I thought that `appFolder` was just going to be the dirname of `execPath`. I'm not sure that `appFolder` communicates that this is the cwd of the the VS Code executable. How about `execCwd`?

Reply:
exec lives in <install path>/bin/, we need <install path> such that we can get to the extensions path from it

Reply:
Also "cwd" is ambiguous imo

Reply:
And dirname appears to be used as the actual directory name, not the directory _path_.

Reply:
Dirname, folder, root is used in the variables, it's a bit of a mess.